oak_functions/wasm/src/lib.rs

```diff
@@ -37,20 +37,27 @@ use oak_functions_extension::{ExtensionFactory, OakApiNativeExtension};
 use oak_logger::{Level, OakLogger};
 use wasmi::{MemoryType, Store};
 
+/// Fixed name of the function to start a Wasm. Every Oak Wasm module has to provide this function.
```

Comment:
```suggestion
/// Fixed name of the function to start a Wasm. Every Oak Wasm module must provide this function.
```

as per https://www.ietf.org/rfc/rfc2119.txt (and below)